src/test/test-models/good/lang/reject_mult_args.stan

```diff
@@ -1,13 +1,13 @@
 functions {
-  real relative_diff(real x, real y, real max, real min) {
+  real relative_diff(real x, real y, real max_v, real min_v) {
```

Comment:
It seems that stanc2 allowed identifiers in function names to shadow math functions, but variable names were not allowed to shadow.

stanc3 is strict on this so had to change here. This is technically breaking backward compatibility, though I honestly find it weird to allow it only for function names. 

https://github.com/stan-dev/stanc3/pull/569 will allow shadowing for all identifiers if its approved.